web/core/src/jsTest/kotlin/elements/EventTests.kt

```diff
@@ -36,6 +39,165 @@ class EventTests {
         assertTrue(handeled)
     }
 
+    @Test
+    fun multipleEventListenersCanBeAddedAndUsed() = runTest {
+        var handled1 = false
+        var handled2 = false
+
+        composition {
+            Button(
+                {
+                    onClick { handled1 = true }
+                    onClick { handled2 = true }
+                }
+            ) {}
+        }
+
+        assertEquals(1, root.childElementCount)
```

Comment:
I mean, it's a test,  it still will fail if we don't have element for some reason - let's discuss whether we need this check at all - it's not like there's smartcast somewhere later on.